analysis.py

```diff
@@ -21,9 +21,12 @@
 
 useful_fields = [
   ['admin_date', ['emd', 'admin', 'current_status', 'date']],
-  # Add cols to pick out here
+  ['admin_date', ['emd', 'admin', 'current_status', 'date']],
+  ['grid', ['emd', 'structure_determination_list', 'structure_determination', 'specimen_preparation_list', 'single_particle_preparation', 'grid', 'model']],
+  ['grid_material', ['emd', 'structure_determination_list', 'structure_determination', 'specimen_preparation_list', 'single_particle_preparation', 'grid', 'material']],
+  ['admin_date', ['emd', 'admin', 'current_status', 'date']],
+  # Add cols to pick out here'
```

Comment:
You will also need to update the SQL file. I was thinking of making a function that can add the correct fields in the SQL file as well, but you'll have to also specify the datatype as detecting it will be too much overhead.